### DISCUSSION TITLE
ZTE K540 does not use Ingenic SoC

### DISCUSSION THREAD

**Discussion by NarkyAbyss** [themactep/thingino-firmware#375] [General]
The K540 does not use Ingenic at all. All chips do not match.
SoC is Sigmastar SSC337DE
Wifi is MT7601UN

@jimsmt I saw you wrote the pull request. which SKU are you using?
I found K540, K540A and K540 v2

![63dede55-6ef5-46aa-96d4-e6ff068a155c](https://github.com/user-attachments/assets/5fe0c30f-099d-4768-9c8e-5a0baecbbc91)
![f399f0ef-e318-4604-848c-7bac5c4521b6](https://github.com/user-attachments/assets/73928d5a-6476-4d5d-bd9f-9667a15cbd72)
![efdf54de-55fc-40df-b088-1b370fb83301](https://github.com/user-attachments/assets/93176641-f440-4f00-94a9-0f70a7d689a9)
![PXL_20241213_173049469](https://github.com/user-attachments/assets/4b774971-161a-4fdf-8d5d-9e6dd2bb91e6)
![PXL_20241213_173221778](https://github.com/user-attachments/assets/ea19357c-9cec-4e78-badf-a455182671e7)
![PXL_20241213_173435685](https://github.com/user-attachments/assets/e67c2690-09a7-4321-a699-5883740569b7)

**Comment by gtxaspec** [comment]
moved to discussions for visibility...

**Comment by NarkyAbyss** [comment]
shall I suggest K540 support be marked as partial support until we can figure out if there was a chip swap or new SKU?

**Reply by gtxaspec** [reply]
Probably different revisions, with a redesign In between. Can't swap to a different vendor/platform chip.

**Reply by NarkyAbyss** [reply]
I still think it makes sense to make a note somewhere for people who have this camera to double check the chip

**Comment by NarkyAbyss** [comment]
*moving to separate thread

Can anyone else give details of their board?
@jimsmt would you mind checking the manufacture date on your unit?

As you can see, this unit shows 2022/01 and board from 2021-04-30. That's a couple years before the pull request
